### ISSUE TITLE
Repl

### ISSUE THREAD

**Issue by EyaadM** [Altoid0/AP-CSA#8]
-[x] Create Repl for project
-[x] Add some frqs/minilabs to project
-[x] Organize Repl through folders
-[ ] Embed REPL project into html with Tanay

**Comment by Altoid0** [comment]
Awesome job on the Repl. Lets add method calls to the main.java file so when people run the project they can select which unit they want to run and it will run all the methods in that unit.

**Comment by tonyhieu** [comment]
I agree with Tanay; without a main method, the user cannot interact with any of the code. I suggest adding more comments to your code in order to make the code clearer.
